packages/cli-platform-apple/src/commands/runCommand/createRun.ts

```diff
@@ -126,7 +127,26 @@ const createRun =
     );
 
     if (platformName === 'macos') {
-      buildProject(xcodeProject, platformName, undefined, mode, scheme, args);
+      const buildOutput = await buildProject(
+        xcodeProject,
+        platformName,
+        undefined,
+        mode,
+        scheme,
+        args,
+      );
+
+      installApp(
```

Comment:
Wdyt about this function accepting an object? Now we need to pass `undefined` with positional params